src/lib/entities/audio/model/audio-instance.ts

```diff
@@ -1,115 +1,23 @@
-import { writable, derived } from 'svelte/store';
+import { writable } from 'svelte/store';
 
-export const audio = writable<HTMLAudioElement | undefined>(undefined);
+// Because apparently Svelte thinks it knows better how the Web APIs should be.
+export type SvelteTimeRanges = Array<{ start: number; end: number }>;
```

Comment:
I don't understand why can't we use the built-in `TimeRanges` everywhere? It has a public constructor we can use for the default values (unlike `FileList`)